client/my-sites/checkout/composite-checkout/components/checkout-main.tsx

```diff
@@ -722,26 +728,34 @@ export default function CheckoutMain( {
 				theme={ theme }
 				selectFirstAvailablePaymentMethod
 			>
-				<WPCheckout
-					loadingContent={
-						<CheckoutLoadingPlaceholder checkoutLoadingConditions={ checkoutLoadingConditions } />
-					}
-					customizedPreviousPath={ customizedPreviousPath }
-					isRemovingProductFromCart={ isRemovingProductFromCart }
-					areThereErrors={ areThereErrors }
-					isInitialCartLoading={ isInitialCartLoading }
-					addItemToCart={ addItemAndLog }
-					changePlanLength={ changePlanLength }
-					countriesList={ countriesList }
-					createUserAndSiteBeforeTransaction={ createUserAndSiteBeforeTransaction }
-					infoMessage={ infoMessage }
-					isLoggedOutCart={ !! isLoggedOutCart }
-					onPageLoadError={ onPageLoadError }
-					removeProductFromCart={ removeProductFromCartAndMaybeRedirect }
-					showErrorMessageBriefly={ showErrorMessageBriefly }
-					siteId={ updatedSiteId }
-					siteUrl={ updatedSiteSlug }
-				/>
+				<>
```

Comment:
In theory, you should be able to have multiple sibling children of `CheckoutProvider` without needing to add this fragment, but I'm guessing you did it because there was a problem. If that's so, what was the issue that this solved?

Reply:
Yeah. It did work - https://github.com/Automattic/wp-calypso/pull/78955/commits/6ebd952eeb69ec8e94332682268322499ad30496